webapi/data/config.ui

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0"?>
```

Comment:
If there are no differences with `config.glade` you can just rename the old file.

Reply:
There are significant differences between the .glade and the .ui file (widget to object nodes in the XML). I'd prefer to keep the ui file as is. I can remove the .glade file as it is no longer needed if we remove the GTK2 code.

Reply:
Ok, let's remove the old one.